src/Deposit.js

```diff
@@ -1096,22 +1117,37 @@ export default class Deposit {
     )
   }
 
-  // Given a Bitcoin transaction and the number of confirmations that need to
-  // be proven constructs an SPV proof and returns the raw parameters that
-  // would be given to an on-chain contract.
-  //
-  // These are:
-  // - version
-  // - txInVector
-  // - txOutVector
-  // - locktime
-  // - outputPosition
-  // - merkleProof
-  // - txInBlockIndex
-  // - chainHeaders
-  //
-  // Constructed this way to serve both qualify + mint and simple
-  // qualification flows.
+  /**
+   * Given a Bitcoin transaction and the number of confirmations that need to
+   * be proven constructs an SPV proof and returns the raw parameters that
+   * would be given to an on-chain contract.
+   *
+   * These are:
+   * - version
+   * - txInVector
+   * - txOutVector
+   * - locktime
+   * - outputPosition
+   * - merkleProof
+   * - txInBlockIndex
+   * - chainHeaders
+   *
+   * Constructed this way to serve both qualify + mint and simple
+   * qualification flows.
+   *
+   * @param {{ transactionID: string, outputPosition: number }} bitcoinTransaction
+   *        The transaction id to construct the proof for and the output
+   *        position of interest.
+   * @param {number} confirmations The number of confirmations that the proof
+   *        should show the given transaction has received. Must be >= the
+   *        number of confirmations the transaction has already received.
+   *
+   * @return {Promise<[Buffer,Buffer,Buffer,Buffer,number,Buffer,number,Buffer]>}
```

Comment:
That's a fun one

Reply:
At what point do one's eyes start swirling 🌀🌀